src/controllers/movie.js

```diff
@@ -0,0 +1,108 @@
+const {
+    getMoviesDB,
+    createMovieDB,
+    getMovieByIdDB,
+    getMovieByTitleDB,
+    updateMovieDB,
+} = require("../domains/movie.js");
+const { get } = require("../routers/movie.js");
+
+const getMovies = async (req, res) => {
+    const filter = {
+        title: req.query.title,
+        runtimeMins: req.query.runtimeMins,
+        runtimeLt: req.query.runtimeLt,
+        runtimeGt: req.query.runtimeGt,
+    };
+    console.log(filter);
```

Comment:
Be careful with this, try not to commit debug logs

Reply:
Would commenting out be good enough or should they be removed all together?